src/components/sky-nav/sky-nav.twig

```diff
@@ -0,0 +1,40 @@
+{# https://webaim.org/techniques/skipnav/ #}
+<a href="#main">
```

Comment:
I wonder if it would be worth adding content within a `<main id="main>` element for the demo? Or will we be creating a page content pattern down the line that will showcase that behavior?